spec/es_spec_helper.rb

```diff
@@ -26,7 +27,7 @@ def get_port
   end
 
   def get_client
-    Elasticsearch::Client.new(:host => "#{get_host}:#{get_port}")
+    Elasticsearch::Client.new(:host => "#{get_host_port}:#{get_port}")
```

Comment:
You don't need the `:#{get_port}` part here, do you?  I see that `get_host_port` does that already.